hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/metrics/OzoneManagerDoubleBufferMetrics.java

```diff
@@ -58,6 +59,9 @@ public class OzoneManagerDoubleBufferMetrics {
       "iteration")
   private MutableGaugeFloat avgFlushTransactionsInOneIteration;
 
+  @Metric(about = "DoubleBuffer queue size.", valueName = "Size")
+  private MutableStat queueSize;
```

Comment:
Just one question: this metric tracks the number of transactions flushed in the last iteration, so it tracks queue size in "batches", whenever flush happens.  Should we also add another metric to track the queue size as items are being added to the `currentBuffer`?  Or would that be redundant?

Reply:
It's a good point. I think the purpose is to monitor the trend of this buffer usage, and the queue size at flushes (or peak queue size) is enough to track this trend. 
```
        flush      
          ↓           ↓   ↓
1 2 3 4 5 6 0 1 2 3 4 5 0 1 0 
```

The intermediate size metric would be helpful if the flushes couldn't happen for any reason or take a long time to complete (in both cases we'll see the intermediate queue size climbing). Yet, it would be a much greater load of datapoints but maybe it's ok because we won't have many OM instances.

Reply:
> much greater load of datapoints

Fair enough.